docs/src/main/tut/docs/patterns/logging/README.md

```diff
@@ -17,7 +17,7 @@ libraryDependencies += "com.47deg" %% "freestyle-logging" % "0.1.0"
 
 ### Operations
 
-The set of abstract operations of the `Logging` algebra are specified as follows:
+The set of abstract operations of the [`Logging` algebra](../../../../../../../freestyle-logging/shared/src/main/scala/logging.scala) are specified as follows:
```

Comment:
Run locally, it fails. It is intended to be use in github, as a link to the source code.